controller/userController.js

```diff
@@ -0,0 +1,61 @@
+const jwt = require('jsonwebtoken');
+
+const User = require('../model/userSchema');
+
+exports.login = async (req, res) => {
+    const username = req.body.username;
+    const password = req.body.password;
+    try {
+        const user = await User.findOne({ username: username });
+        if (!user) {
+            res.status(401).json({ err: "A user with this email could not be found." });
+        }
+        if (user.password !== password)
```

Comment:
password should not be compared directly, instead you need to use `bcrypt` npm module to compare passwords.
You should have hashed version of password stored to the db, whereas you receive plain text password in `req.body`.
Please check this doc about bcrypt usage:
https://www.loginradius.com/blog/engineering/hashing-user-passwords-using-bcryptjs/